superset/connectors/base/models.py

```diff
@@ -185,6 +186,34 @@ def data(self):
             'verbose_map': verbose_map,
         }
 
+    @staticmethod
+    def filter_values_handler(
+            values, target_column_is_numeric=False, is_list_target=False):
+        def handle_single_value(v):
+            # backward compatibility with previous <select> components
+            if isinstance(v, basestring):
+                v = v.strip().strip("'").strip('"')
```

Comment:
You can do a single strip to remove whitespace and single/double quotes:

```python
v = v.strip(' \'"')
```

Or, to also take care of tabs and line feeds:

```python
v = v.strip('\t\n \'"')
```

Reply:
oh neat